src/js/client/binder.js

```diff
@@ -0,0 +1,113 @@
+// Add persistent bindings between a selector and a model value.  Changes are propagated between the two.
+//
+// To use this, you should have a "bindings" element in your fluid defaults, as in:
+//      bindings: [{
+//          selector:    "input",
+//          path:        "input",
+//          elementType: "text",
+//          cookify:     true
```

Comment:
I don't see that this flag is acted on anywhere, nor is it documented - also, I don't think it belongs here